File: apps/src/main/kotlin/io/dustin/apps/common/config/WebConfig.java
//package io.dustin.apps.common.config;
//
//import io.dustin.apps.common.resolver.CustomTestResolver;
//import org.springframework.context.annotation.Bean;
//import org.springframework.context.annotation.Configuration;
//import org.springframework.web.method.support.HandlerMethodArgumentResolver;
//import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
//
//import java.util.List;
//
//@Configuration
//public class WebConfig implements WebMvcConfigurer {
//
//    @Override
//    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers) {
//        resolvers.add(customTestResolver());
//    }
//
//    @Bean
//    public CustomTestResolver customTestResolver() {
//        return new CustomTestResolver();
//    }
//
//}

File: JavaToKotlin/apps(kotlin)/src/main/kotlin/io/dustin/apps/common/config/WebConfig.kt
package io.dustin.apps.common.config

import io.dustin.apps.common.resolver.CustomTestResolver

@Configuration
class WebConfig : WebMvcConfigurer {
    @Override
    fun addArgumentResolvers(resolvers: List<HandlerMethodArgumentResolver?>?) {
        resolvers.add(customTestResolver())
    }

    @Bean
    fun customTestResolver(): CustomTestResolver? {
        return CustomTestResolver()
    }
}
